### ISSUE TITLE
Include Text File of Glyph Sources

### ISSUE THREAD

**Issue by robloo** [unoplatform/uno.fonts#16]
## What would you like to be added:

How was this font built? Was it done entirely manually in FontForge? Was a script used? 

My intention is to obsolete this repository and generate a new, more maintainable symbols font :) However, it would save a lot of time if someone kept track of source glyphs. If they did, would be great to get an MIT licensed file I can process further.

## Why is this needed:

Make glyph source/destination mapping clear and usable in other tooling.

## For which Platform:

- [ ] iOS
- [ ] Android
- [ ] WebAssembly
- [ ] WebAssembly renders for Xamarin.Forms
- [ ] Windows
- [ ] Build tasks

**Comment by davidjohnoliver** [comment]
#17 includes the hard info I was able to gather, including the Fluent System Icons assets used and the glyph positions they were mapped to. It also provides raw assets for the original icons that were created for glyphs that are used in the WinUI Fluent styles for which there was no equivalent in Fluent System Icons (at least at the time).

As to the question of how the font was created: it was indeed done manually with FontForge. 

It seems that the WinJS Symbols font was used as a base. I'm not sure if this would've been the [original](https://github.com/microsoft/fonts/tree/master/Symbols), or if it was [the version used by Uno.UI](https://github.com/unoplatform/uno/blob/master/src/SolutionTemplate/UnoSolutionTemplate.VSIX/winjs-symbols.ttf) prior to Uno.Fonts, which would've had existing minor modifications. (If I had to guess, I would guess the latter.)

In addition, some icons were sourced from the [Icons8 Line Awesome](https://github.com/icons8/line-awesome) icon set (MIT license), but unfortunately no record was kept of which ones.

Finally there were a couple of adjustments made to fix visual issues with individual controls, going from the [commit history](https://github.com/unoplatform/uno.fonts/commits/master).

**Comment by robloo** [comment]
> It seems that the WinJS Symbols font was used as a base

No problem, that is already supported in my SymbolIconManager.

> In addition, some icons were sourced from the Icons8 Line Awesome icon set (MIT license), but unfortunately no record was kept of which ones.

That shouldn't be an issue. I can add that as an IconSet source in the future.

> Finally there were a couple of adjustments made to fix visual issues with individual controls, going from the commit history.

This is more problematic. I would hope their are better icons we can use now days that don't require tweaks. However, the mapping file format I am using [here](https://github.com/robloo/SymbolIconManager/blob/main/Source/Data/Mappings/Example.json) could be extended to support python commands to transform glyphs as needed. This should allow almost full customization on a per-glyph basis (move left/right, adjust width, etc.). That won't do anything if source SVG needs modification though. Custom icon sources could probably also be added to fix this.

**Comment by davidjohnoliver** [comment]
> This is more problematic. I would hope their are better icons we can use now days that don't require tweaks. However, the mapping file format I am using here could be extended to support python commands to transform glyphs as needed. This should allow almost full customization on a per-glyph basis (move left/right, adjust width, etc.). That won't do anything if source SVG needs modification though. Custom icon sources could probably also be added to fix this.

From memory, it's not really about the icons being 'better' or 'worse' in a vacuum. There's a couple of cases where the glyph has to be pixel-perfect matched with Segoe or it looks 'off' - eg [check marks in CheckBox](https://github.com/unoplatform/uno/issues/3664). Custom icon sources can probably be a good way to go.

**Comment by robloo** [comment]
@davidjohnoliver Makes sense. If Uno is willing to donate the source SVG files under MIT I will pull them into a custom 'Undefined' icon set. It won't take long at all to support these. Matching will be done by fileName <-> iconName and mapping file format doesnt need to change.
